scripts/release/asset_publish.py

```diff
@@ -138,6 +138,57 @@ def prepare_model(model_config: assets.ModelConfig, spec_file_path: Path, model_
     return can_publish_model
 
 
+def validate_command_component(component: Component, spec_path: Path, final_version: str) -> bool:
```

Comment:
Consider a more descriptive name, to make it clear that this function changes the spec files:
```suggestion
def validate_update_command_component(component: Component, spec_path: Path, final_version: str) -> bool:
```

Reply:
Done